tests/testthat/test-build-article.R

```diff
@@ -0,0 +1,20 @@
+context("build_article")
+
+test_that("safe_file_copy (used in render_rmarkdown) copies files in subdirs", {
```

Comment:
I think it would be better to create a more holistic test of `render_rmarkdown()`, since it's likely that we will need more in future. Can you please create a simple `.Rmd` file that illustrates the problem?